adcm_aio_client/core/objects/_common.py

```diff
@@ -56,3 +57,10 @@ class WithJobStatus(WithProtectedRequester, AwareOfOwnPath):
     async def get_job_status(self: Self) -> JobStatus:
         response = await self._requester.get(*self.get_own_path())
         return JobStatus(response.as_dict()["status"])
+
+
+class WithImports(WithProtectedRequester, AwareOfOwnPath):
+    @async_cached_property
+    async def imports(self: Self) -> Imports:
+        response = await self._requester.get(*self.get_own_path(), "imports")
+        return Imports(self._requester, self.get_own_path(), response.as_dict()["results"][0])
```

Comment:
path is incorrect, there should be "imports"

Reply:
Path is still incorrect, it should use both `get_own_path` of parent and `"imports"` part

Reply:
changed

Reply:
here it is, but not in `response = await self._requester.get("imports")`, use `_path`

Reply:
replaced